docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -129,6 +129,29 @@ In Sourcegraph 5.2 and earlier, you should use the feature flag `cody` to turn C
 />
 
 
+## Use StarCoder for Autocomplete
+
+Over the past months, we've run extensive tests comparing many coder models for the Autocomplete use case and found that StarCoder offers significant improvements in both quality as well as latency for our users on Sourcegraph.com. You can read about the improvements in [October 2023 release notes](https://sourcegraph.com/blog/feature-release-october-2023) and the [GA release notes](https://sourcegraph.com/blog/cody-is-generally-available).
+
+To be able to offer our Enterprise users the best experience for Cody Autocomplete, we have partnered with [Fireworks](https://fireworks.ai/) and have set up an enterprise-grade deployment with dedicated hardware capacity. Since Sourcegraph 5.3, every Cody setup that uses Cody Gateway is able to opt-in to using [StarCoder](https://huggingface.co/blog/starcoder) as an alternative to Claude Instant as the AI model for Autocomplete.
```

Comment:
```suggestion
To get the best Cody autocomplete experience, Enterprise users can set up an enterprise-grade deployment with dedicated hardware with [Fireworks](https://fireworks.ai/). Since Sourcegraph 5.3, every Cody setup that uses [Cody Gateway](/cody/core-concepts/cody-gateway) can opt-in to using [StarCoder](https://huggingface.co/blog/starcoder) as an alternative to Claude Instant as the AI model for autocomplete.
```